gradle/dependencies.gradle

```diff
@@ -72,6 +72,7 @@ versions += [
   httpclient: "4.5.8",
   easymock: "4.0.2",
   jackson: "2.9.9",
+  jacksonDatabind: "2.9.9.2",
```

Comment:
Question on this addition: Instead of hardcoding this version here, perhaps it might be nicer to add a section after this array definition:
```
# In most cases we want the jackson-databind version to match jackson, unless 
# there's an updated micropatch
# versions["jacksonDatabind"] = versions["jackson"]
versions["jacksonDatabind"] = "2.9.9.2"
```

That way during the next upgrade to jackson, we mirror the versions, and future jackson upgrades only need to update the main version.

Reply:
I think that's more likely to cause issues as the two jackson library versions are not near each other.